megamek/src/megamek/common/Aero.java

```diff
@@ -128,6 +128,13 @@
     // ignored crew hit for harjel
     private int ignoredCrewHits = 0;
     private int cockpitType = COCKPIT_STANDARD;
+    
+    //Autoejection
+    private boolean autoEject = true;
+    private boolean condEjectAmmo = true;
+    private boolean condEjectFuel = true;
+    private boolean condEjectSIDest = true;
+    private boolean condEjectCTDest = false; //LAM only
```

Comment:
The value of this field is never used and is irrelevant for this class.

Reply:
True, but it's part of the IAero interface since LAMs need it and Eclipse won't let me take it out of Aero.

Reply:
Fixed